docker-compose.yml

```diff
@@ -48,11 +48,38 @@ services:
       - 6001:6000
     depends_on:
       - yaba_api
+  
+  yaba_visualization:
```

Comment:
Would it make sense to set environment variables here that reference the other YABA server addresses that can be accessed through ReactJS's `process.env`? This way only one place needs to change if the server information changes

Reply:
Yeah, that's a great idea. Here a new issue for that #72 .